modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/Operation.java

```diff
@@ -134,4 +134,47 @@ public Message installApplication(ApplicationWrapper applicationWrapper) throws
             throw new MDMAPIException(msg, e);
         }
     }
+
+	@POST
+	@Path("uninstallApp/{tenantDomain}")
```

Comment:
This is added to fulfil a requirement which is requested by AppM. The reason to add tenantDomain in here is to remove specific set of apps to an entire tenant as a bulk operation.